lib/src/Base/Func/openturns/IndicatorEvaluation.hxx

```diff
@@ -63,18 +60,7 @@ public:
   /** Operator () */
   using EvaluationImplementation::operator();
```

Comment:
Is this line still necessary, now that ` Sample IndicatorEvaluation::operator() (const Sample & inSample)` has been overridden?